### PR TITLE
(PiCT) Security, session tokens last too long

### PR DESCRIPTION
**Pull request by adisa39**
authentication web token modified to expire in 1day as suggested by the PiCT.
please only merge the changes in *"src/helpers/jwt.ts"* to avoid conflict with main.

### CODE REVIEW

File: src/helpers/jwt.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const generateUserToken = (user: IUser) => {
   try {
     logger.info(`Generating token for user: ${user.pi_uid}`);
     const token = jwt.sign({ userId: user.pi_uid, _id: user._id }, env.JWT_SECRET, {
-      expiresIn: "30d",
+      expiresIn: "1d",
```
Comment (swoocn) [inline src/helpers/jwt.ts:13]: Hi @adisa39 - shouldn't we also expire the cookie to 1 day as well back in the userController.ts?

Comment (swoocn) [reply src/helpers/jwt.ts:13]: I also take it that you'll be building the corresponding frontend implementation as well.

Comment (adisa39) [reply src/helpers/jwt.ts:13]: > Hi @adisa39 - shouldn't we also expire the cookie to 1 day as well back in the userController.ts?

yes we should. though we are not using cookies right now but the 30 should also be changed to 1

Comment (adisa39) [reply src/helpers/jwt.ts:13]: > I also take it that you'll be building the corresponding frontend implementation as well..?

there is no frontend implementation for this, it's all handled just with this. When a user token expired, another token will be generated automatically provided the pioneer is found.

**Review by swoocn — APPROVED** [review]
LGTM after reverting back to recent changes + cookies expiration mod. 👍
Build is stabilized and ready for deployment.

**Pull request merged**